docs/04-operation-guides/operations/10-backup-kyma.md

```diff
@@ -1,29 +1,232 @@
+
+## Create on-demand volume snapshots
+
+If you want to provision a new volume or restore the existing one, create on-demand [volume snapshots](https://kubernetes.io/docs/concepts/storage/volume-snapshots/):
+
+<div tabs name="backup-providers">
+  <details>
+  <summary label="AKS">
+  AKS
+  </summary>
+
+### Prerequisites
+
+  The minimum supported Kubernetes version is 1.17.
+
+### Steps
+
+  1. [Install the CSI driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/install-csi-driver-master.md).
+  2. [Create a volume snapshot](https://github.com/kubernetes-sigs/azuredisk-csi-driver/tree/master/deploy/example/snapshot).
+
+  </details>
+  
+  <details>
+  <summary label="GKE">
+  GKE
+  </summary>
+
+### Prerequisites
+
+  The minimum supported Kubernetes version is 1.14.
+
+### Steps
+
+  1. [Enable the required feature gate on the cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/gce-pd-csi-driver).
+  2. Check out [the repository for the Google Compute Engine Persistent Disk (GCE PD) CSI driver](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) for details on how to use volume snapshots on GKE.
+
+  </details>
+
+  <details>
+  <summary label="Gardener GCP">
+  Gardener
+  </summary>
+
+### Prerequisites
+
+  - As of Kubernetes version 1.18, Gardener **GCP** and **AWS** by default use CSI drivers and support taking volume snapshots.
+  - Gardener **Azure** supports CSI drivers as of Kubernetes version 1.21.
+
+### Steps
+
+  1. Create a VolumeSnapshotClass with the correct driver:
+    - for GCP: `pd.csi.storage.gke.io`
+    - for AWS: `ebs.csi.aws.com`
+    - for Azure: `disk.csi.azure.com`
+
+  ```yaml
+  apiVersion: snapshot.storage.k8s.io/v1beta1
+  kind: VolumeSnapshotClass
+  metadata:
+    annotations:
+      snapshot.storage.kubernetes.io/is-default-class: "true"
+    name: snapshot-class
+  driver: <enter correct one for cloud provider>
+  deletionPolicy: Delete
+  ```
+  
+  2. Create a VolumeSnapshot resource:
+
+  ```yaml
+  apiVersion: snapshot.storage.k8s.io/v1beta1
+  kind: VolumeSnapshot
+  metadata:
+    name: snapshot
+  spec:
+    source:
+      persistentVolumeClaimName: {PVC_NAME}
+  ```
+
+  3. Wait until the **READYTOUSE** field has the `true` status to verify that the snapshot was taken successfully:
+
+  ```bash
+  kubectl get volumesnapshot -w
+  ```
+
+  4. Use this snapshot as a datasource to create a PVC:
+  
+  ```yaml
+  apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: pvc-restored
+  spec:
+    accessModes:
+     - ReadWriteOnce
+    storageClassName: snapshot-class
+    resources:
+      requests:
+        storage: {SIZE_OF_ORIGINAL_PVC}
+    dataSource:
+      name: snapshot
+      kind: VolumeSnapshot
+      apiGroup: snapshot.storage.k8s.io
+  ```
+
+  </details>
+ </div>
+
+## Create periodic snapshot job
```

Comment:
```suggestion
## Create a periodic snapshot job
```
Jobs always seemed countable to me, what do you think?

Reply:
Fully agree, thanks for spotting this!